code/modules/cargo/bounties/assistant.dm

```diff
@@ -1,20 +1,20 @@
 /datum/bounty/item/assistant/strange_object
 	name = "Strange Object"
 	description = "Nanotrasen has taken an interest in strange objects. Find one in maint, and ship it off to CentCom right away."
-	reward = 600
+	reward = 2200
```

Comment:
1000 at most, these things spawn all over the place and should not be worth as much as taser crates